enforcer_synced.go

```diff
@@ -418,3 +418,45 @@ func (e *SyncedEnforcer) AddNamedPolicies(ptype string, rules [][]string) (bool,
 	defer e.m.Unlock()
 	return e.Enforcer.AddNamedPolicies(ptype, rules)
 }
+
+// GetImplicitUsersForPermission gets implicit users for a permission.
+// For example:
+// p, admin, data1, read
+// p, bob, data1, read
+// g, alice, admin
+//
+// GetImplicitUsersForPermission("data1", "read") will get: ["alice", "bob"].
+// Note: only users will be returned, roles (2nd arg in "g") will be excluded.
+func (e *SyncedEnforcer) GetImplicitPermissionsForUser(user string, domain ...string) ([][]string, error) {
```

Comment:
The documentation is for `GetImplicitUsersForPermission` but the function is called `GetImplicitPermissionsForUser`